recommendations/forms.py

```diff
@@ -39,6 +37,17 @@ class RuntimeForm(forms.Form):
     )
 
     OPTIONS = RUNTIME_SPANS
+    Runtimes = forms.ChoiceField(widget=forms.RadioSelect, choices=OPTIONS)
+
+
+class LanguageForm(forms.Form):
+    Question = forms.CharField(
+        initial="Pick some languages! Pick at least three.",
+        disabled=True,
```

Comment:
Is this meant to be disabled? After the language selection my movie choices seemed to always drop to 0.